internal-packages/workflow-designer-ui/src/editor/properties-panel/text-generation-node-properties-panel/tools/tool-provider/github/index.tsx

```diff
@@ -540,7 +540,7 @@ function GitHubToolConfigurationDialogInternal({
 												tool.toLowerCase(),
 											)}`}
 											target="_blank"
-											rel="noopener"
+											rel="noopener noreferrer"
```

Comment:
```suggestion
											rel="noopener"
```

Reply:
https://docs.giselles.ai is a domain that we manage, so there is no problem if you do not add noreferrer.